src/GafferSceneUI/SceneView.cpp

```diff
@@ -740,6 +740,22 @@ class CameraOverlay : public GafferUI::Gadget
 			return m_cropWindow;
 		}
 
+		// left, top, right, bottom
+		void setOverscan( const V4f &overscan )
```

Comment:
Could you remove the todo at line 677 please? As things are panning out, this is becoming a dedicated CameraOverlay class that we won't try to generalise, and the `CropWindowTool::Rectangle` thingy will get refactored into a Handle of some sort, which definitely won't have stuff for overscan etc.

Reply:
Removed in https://github.com/GafferHQ/gaffer/pull/3584/commits/b03f024644c37c6ac645c58c0799122bc557a6d6